odoo/addons/base/tests/test_acl.py

```diff
@@ -28,15 +28,39 @@ def test_field_visibility_restriction(self):
            field for users who do not belong to one of the explicitly allowed groups"""
         currency = self.env['res.currency'].with_user(self.demo_user)
 
+        # Add a view that adds a label for the field we are going to check
+        extension = self.env["ir.ui.view"].create({
+            "name": "Add separate label for decimal_places",
+            "model": "res.currency",
+            "inherit_id": self.env.ref("base.view_currency_form").id,
+            "arch": """
+                <data>
+                    <field name="decimal_places" position="attributes">
+                        <attribute name="nolabel">1</attribute>
+                    </field>
+                    <field name="decimal_places" position="before">
+                        <label for="decimal_places"/>
+                    </field>
+                </data>
+            """,
+        })
+        currency = currency.with_context(check_view_ids=extension.ids)
+
         # Verify the test environment first
         original_fields = currency.fields_get([])
         form_view = currency.fields_view_get(False, 'form')
         view_arch = etree.fromstring(form_view.get('arch'))
         has_group_system = self.demo_user.has_group(GROUP_SYSTEM)
         self.assertFalse(has_group_system, "`demo` user should not belong to the restricted group before the test")
         self.assertIn('decimal_places', original_fields, "'decimal_places' field must be properly visible before the test")
+<<<<<<< HEAD
         self.assertNotEqual(view_arch.xpath("//field[@name='decimal_places']"), [],
+=======
+        self.assertNotEquals(view_arch.xpath("//field[@name='decimal_places'][@nolabel='1']"), [],
```

Comment:
This should be the good one.

Reply:
I also replaced assertEquals by assertEqual, seem the one to use in python3.